src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -224,8 +224,11 @@ module.exports.commands = [{
         self.waitForElementVisible('@tests');
         return self;
     },
-    clickFirstResultItem: function (expand = true) {
+    clickFirstResultItem: function (expand) {
         var self = this;
+        if (!expand) {
+            expand = true;
+        }
```

Comment:
I know there were definitely usages of this method where the param was passed as true, false and omitted altogether.